src/tests/index.test.ts

```diff
@@ -1,19 +1,15 @@
 import 'jest';
 
-/* eslint-disable @typescript-eslint/no-var-requires, global-require */
+/* eslint-disable @typescript-eslint/no-var-requires, global-require, no-console */
+
+// console.warn('WARNING: this test will not pass if port 3000 is in use!');
 
 const testPort = 3212;
 
 describe('Server', () => {
   beforeAll(() => {
-    global.console = {
-      ...global.console,
-      log: jest.fn(), // console.log are ignored in tests
-      //   error: jest.fn(),
-      //   warn: jest.fn(),
-      //   info: jest.fn(),
-      //   debug: jest.fn(),
-    };
+    // Hide Bolt API key warnings
+    process.env.LOG_LEVEL = 'emerg';
```

Comment:
Did this work? Because LOG_LEVEL is only listened to by logging libraries. console.log ignores this. I didn't know the reason for this was because of Bolt.

But if it works, then that means they use a logging library, and it just so happens to work 😆 Which I am fine with

Reply:
Sure did! If it ain’t broke...